boards/nucleo-f401/include/periph_conf.h

```diff
@@ -103,6 +102,48 @@ static const uart_conf_t uart_config[] = {
 /** @} */
 
 /**
+ * @brief   PWM configuration
+ * @{
+ */
+static const pwm_conf_t pwm_config[] = {
+    { /* D3 */
+        .dev      = TIM2,
+        .rcc_mask = RCC_APB1ENR_TIM2EN,
+        .pins     = { GPIO_PIN(PORT_B, 3) },
+        .af       = GPIO_AF1,
+        .chan     = 2,
+        .bus      = APB1
+    },
+    { /* D5 */
+        .dev      = TIM3,
+        .rcc_mask = RCC_APB1ENR_TIM3EN,
+        .pins     = { GPIO_PIN(PORT_B, 4) },
+        .af       = GPIO_AF2,
+        .chan     = 1,
```

Comment:
with this configuration, `TIM3` is overloaded -> this would lead to trouble. I think what you actually want is to configure TIM3 with 2 channels:
```c
...
   { /* D5 and D9 */
        .dev      = TIM3,
        .rcc_mask = RCC_APB1ENR_TIM3EN,
        .pins     = { GPIO_PIN(PORT_B, 4), GPIO_PIN(PORT_C, 7), GPIO_UNDEF, GPIO_UNDEF },
        .af       = GPIO_AF2,
        .chan     = 2,
        .bus      = APB1
    },
...
```
Note also, that you always need to map 4 pins, but you may set them to `GPIO_UNDEF` for un-used channels.